roles/common/tasks/main.yaml

```diff
@@ -88,6 +88,24 @@
       when:
         - tfstate.postgres_servers is defined
         - tfstate.postgres_servers.value|length > 0
+    - name: tfstate - ebs csi driver account
+      set_fact: 
+        EBS_CSI_DRIVER_ACCOUNT: "{{ tfstate.ebs_csi_account.value }}"
+      when:
+        - tfstate.ebs_csi_account is defined
+        - tfstate.ebs_csi_account.value|length > 0
+    - name: tfstate - ebs csi driver location
+      set_fact: 
+        EBS_CSI_DRIVER_LOCATION: "{{ tfstate.location.value }}"
+      when:
+        - tfstate.location is defined
+        - tfstate.location.value|length > 0
+    - name: tfstate - k8s server version
+      set_fact: 
+        K8S_SERVER_VERSION: "{{ tfstate.k8s_server_version.value }}"
```

Comment:
Why `k8s_server_version`? This is the kubernetes version. `k8s_version`

Reply:
sure, I'll update that

Reply:
updated to k8s_version as per review comment